plugins/extractors/maxcompute/maxcompute.go

```diff
@@ -209,6 +243,12 @@ func (e *Extractor) buildAsset(schema *odps.Schema, _ *odps.Table, tableType str
 		UpdateTime: timestamppb.New(time.Time(tableSchema.LastModifiedTime)),
 	}
 
+	maxPreviewRows := e.config.MaxPreviewRows
+	if maxPreviewRows != -1 {
```

Comment:
better to change  as `maxPreviewRows > 0` to prevent zero or another negative value

Reply:
okay, addressed